clang/test/Driver/sycl-spirv-obj.cpp

```diff
@@ -0,0 +1,36 @@
+///
+/// Tests for SPIR-V related objects
```

Comment:
Supposing https://github.com/intel/llvm/pull/5251 gets merged first, can a dummy object-based test be crafted for mixed SPIR-V-containing object + LLVM BC-containing object input consumption over the top of `spirv-to-ir-wrapper` changes? Aside from that, I remain of my [earlier opinion](https://github.com/intel/llvm/pull/5251#pullrequestreview-842209166) that the introduction of E2E tests need not block the product changes in this case.

Reply:
Yes - it would be good to have a test the covers that.  This can be added once everything is in place in a separate PR.